.github/workflows/fleximod_test.yaml

```
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    branches: [ development ]
```

I might have this run on all pull requests instead of just development (as we'll likely only ever do direct commits to `main`):

```suggestion
  pull_request:
```

---

done!